tests/layer_tests/tensorflow_tests/test_tf_UnaryOps.py

```diff
@@ -152,6 +158,7 @@ def create_net_with_unary_op(self, shape, ir_version, op_type, use_new_frontend)
                                          'LogicalNot',
                                          'Square',
                                          'Erf',
+                                         'BitwiseNot'
```

Comment:
the same comment

Reply:
Similar case as in binary, in L189 single case is marked as precommit_tf_fe, in addition there is also separate test case with just "precommit" mark that seem to just duplicate second test. Maybe they should be merged together?